libdap2/ncd2dispatch.c

```diff
@@ -2434,6 +2434,15 @@ NCD2_var_par_access(int ncid, int p2, int p3)
     return THROW(NC_ENOPAR);
 }
 
+int
```

Comment:
Was there a reason to move this function in the file? Or was something else done?